Task 1.1.3

Lepert1 opened the pull request.

Lepert1 committed:
Task 1.1.3

Lepert1 committed:
Task 1.1.3
add file .gitignore

pdasilem commented on src/main/java/jm/task/core/jdbc/Main.java:
```diff
@@ -1,7 +1,27 @@
+
 public class Main {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws SQLException {
```
не должно быть здесь исключений

pdasilem commented on src/main/java/jm/task/core/jdbc/dao/UserDaoJDBCImpl.java:
```diff
@@ -1,35 +1,97 @@
     public void createUsersTable() {
+        String sqlQuery = "CREATE TABLE Users (id SERIAL PRIMARY KEY, name varchar(255), lastname varchar(255), age smallint);";
 
+        try {
```
используй ресурсы для закрытия объектов

Lepert1 committed:
Update Main.java

Убрал throws SQLException

Lepert1 committed:
Update UserDaoJDBCImpl.java

добавил использование с ресурсами
try(Statement statement = openConnect().createStatement())

pdasilem commented on src/main/java/jm/task/core/jdbc/dao/UserDaoJDBCImpl.java:
```diff
@@ -1,35 +1,91 @@
+            statement.executeUpdate(sqlQuery);
+            System.out.println("Table deleted");
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```
зачем выбрасываешь исключение здесь и далее?

pdasilem commented on src/main/java/jm/task/core/jdbc/util/Util.java:
```diff
@@ -1,5 +1,38 @@
+            connection.close();
+            System.out.println("Close Database");
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```
зачем здесь выбрасываешь исключение?
